frontend/doc-site/03__Download Published Data.mdx

```diff
@@ -1,6 +1,6 @@
 # Downloading Published Data on CZ CELLxGENE Discover
 
-Clicking the download button launches a dialog that enables a dataset to be downloaded in h5ad (AnnData v0.7) and rds (Seurat v3) formats. All datasets adhere to the CELLxGENE single cell annotated data schema. Datasets can either be downloaded via the browser by clicking the blue download button, or via the command line by pasting the provided curl command.
+Clicking the download button launches a dialog that enables a dataset to be downloaded in h5ad (AnnData v0.8) and rds (Seurat v3) formats. All datasets adhere to the CELLxGENE single cell annotated data schema. Datasets can either be downloaded via the browser by clicking the blue download button, or via the command line by pasting the provided curl command.
 
 1. Select the data set you wish to download.
```

Comment:
The images are not rendering in the file preview, but I would expect that we need new screen captures for the download dialog which will display anndata 0.7 ...

https://cellxgene.cziscience.com/docs/03__Download%20Published%20Data